deploy/examples/create-external-cluster-resources.py

```diff
@@ -826,7 +938,37 @@ def validate_rados_namespace(self):
             raise ExecutionFailureException(
                 ("The provided rados Namespace, '{}', is not found in the pool '{}'").format(
                     rados_namespace, rbd_pool_name))
-
+    
+    def get_rgw_fsid(self):
+        access_key = self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY']
+        secret_key = self.out_map['RGW_ADMIN_OPS_USER_SECRET_KEY']
+        rgw_endpoint = self._arg_parser.rgw_endpoint
+        base_url = "http://"
+        if self._arg_parser.rgw_tls_cert_path:
+            base_url = "https://"
+        base_url = base_url + rgw_endpoint + "/admin/info?"   
+        params = {'format': 'json'}
+        request_url = base_url + urllib.parse.urlencode(params)
+            
+        try:
+            r = requests.get(request_url, auth=S3Auth(access_key, secret_key,rgw_endpoint))
```

Comment:
Does `get()` accept a modified client? How can we pass the certificate or ignore the certificate verification?

Reply:
Yes it can be modified,  changed some code let see if it passes the CI

Reply:
@leseb The changes are working, just need to understand what will the cert value/path that I can provide in the CI

Reply:
We cannot use certs with the CI since they are self-signed. We can however use the insecure TLS option if we deploy RGW with TLS in the CI.

Reply:
I saw the Pr in which you added the TLS and gave the example of `--rgw-tls-cert-path my-cert` https://github.com/rook/rook/commit/d6451527437e8e999c3ea6d38d17f7969e32b989 so if some How I can replicate this behaviour.